codegen/utils.ts

```diff
@@ -0,0 +1,117 @@
+import * as M from "../frame_metadata/mod.ts";
+
+export const importSource = new URL("../mod.ts", import.meta.url).toString();
```

Comment:
How do we want to deal with Capi import specifier across project types?

- a new string param in the CLI?
- attempt to infer based on current process (does `Deno` global exist, misc.)
- ...

Reply:
#258